### PR TITLE
Chicago cubs master update 02

### PR DESCRIPTION
**Pull request by alesstimec**
(Review request: http://reviews.vapour.ws/r/2737/)

### CODE REVIEW

File: juju/series/series_windows_test.go [hunk 1]
```diff
@@ -7,9 +7,11 @@ package series_test
 import (
 	"fmt"
 
+	"github.com/juju/testing"
+	"github.com/juju/utils"
```
Comment (rogpeppe) [inline juju/series/series_windows_test.go]: This won't compile under windows (duplicate import). Run sortimports on this, then (assuming you've got go 1.5), try

GOOS=windows go test -c

in this directory to check that it compiles.

**Comment by rogpeppe** [comment]
LGTM but you'll need to fix the series package tests for Windows to uncurse it.

**Comment by alesstimec** [comment]
$$merge$$

**Comment by alesstimec** [comment]
$$merge$$

**Comment by alesstimec** [comment]
$$merge$$

**Comment by alesstimec** [comment]
$$merge$$

**Pull request merged**